#Base
logger/Event base
#battery status
logger/BatteryStatus battery
# props
uint16 mision_id
